q2_vsearch/_join_pairs.py

```diff
@@ -29,10 +29,6 @@
     'minmergelen': None,
     'maxmergelen': None,
     'maxee': None,
-    'qmin': 0,
-    'qminout': 0,
-    'qmax': 41,
-    'qmaxout': 41,
```

Comment:
I removed the qmin and qminout parameters because these are cases that currently aren't possible to hit, given our available fastq formats in q2-types. In newer versions of vsearch there appears to be some new validation, to prevent invalid usage of these flags, so I suspect these were never really possible for us to use in the first place. Given that I have removed qmin and qminout, it seemed to make sense to remove qmax and qmaxout, too.

Reply:
We could keep these flags set internally and not expose them as user-adjustable parameters, which I think might address this concern.

Reply:
done!